lib/spack/spack/package.py

```diff
@@ -545,6 +545,10 @@ class SomePackage(Package):
     # Verbosity level, preserved across installs.
     _verbose = None
 
+    # List of strings which contains GitHub usernames of package maintainers.
+    # Do not include @ here in order not to unnecessarily ping the users.
```

Comment:
Can you replace `#` with `#:` so that this makes it into the API docs?

I don't think `@mentions` in a file will ping the user, but I agree that there's no point in making it required for the variable.